warehouse/forklift/legacy.py

```diff
@@ -993,10 +993,11 @@ def file_upload(request):
                     raise _exc_with_message(
                         HTTPBadRequest,
                         "File too large. " +
-                        "Limit for project {name!r} is {limit}MB".format(
+                        "Limit for project {name!r} is {limit}MB. ".format(
                             name=project.name,
-                            limit=file_size_limit // (1024 * 1024),
-                        ))
+                            limit=file_size_limit // (1024 * 1024)) +
+                        "See https://pypi.org/help/#file_size_limit",
```

Comment:
Instead of hardcoding this link, could you generate it like `request.route_url('help', _anchor='file-file-size-limit')`, otherwise it will be wrong for TestPyPI.

Reply:
(Also I realize there are a few other places where this isn't being done... perhaps you'd like to fix those too? 🙂)

Reply:
AHA. I tried to figure that out - evidently not hard enough. :) Thanks for the tip. Fixing.